packages/esm-outpatient-app/src/patient-search/visit-form/visit-form.component.tsx

```diff
@@ -218,16 +218,7 @@ const StartVisitForm: React.FC<VisitFormProps> = ({ patientUuid, toggleSearchTyp
           </Row>
         )}
         <div className={styles.backButton}>
```

Comment:
When someone discards the form and tries to add a new queue entry, the visit form is still launched